### PR TITLE
docs: Gatsby packages

### PR DESCRIPTION
**Pull request by kendallstrautman**
Mainly copied over helper info from the guides

### CODE REVIEW

File: packages/gatsby-plugin-tinacms/README.md [hunk 1]
```diff
@@ -36,20 +22,28 @@ module.exports = {
...
 ```
 
-## Next Steps
+## Options
+
+```ts
+export interface GatsbyPluginTinacmsOptions {
+  enabled?: boolean
+  sidebar: TinaCMSConfig['sidebar']
+  toolbar: TinaCMSConfig['toolbar']
```
Comment (ncphillips) [inline packages/gatsby-plugin-tinacms/README.md:44]: Maybe we could add a link to the sidebar/toolbar docs 

https://tinacms.org/docs/ui#enabling-the-user-interface

Comment (kendallstrautman) [inline packages/gatsby-plugin-tinacms/README.md:44]:
```suggestion
  [toolbar: TinaCMSConfig['toolbar']](https://tinacms.org/docs/ui#enabling-the-user-interface)
```

File: packages/gatsby-plugin-tinacms/README.md [hunk 2]
```diff
@@ -36,20 +22,28 @@ module.exports = {
...
 }
 ```
 
-## Next Steps
+## Options
+
+```ts
+export interface GatsbyPluginTinacmsOptions {
+  enabled?: boolean
+  sidebar: TinaCMSConfig['sidebar']
```
Comment (kendallstrautman) [inline packages/gatsby-plugin-tinacms/README.md:43]:
```suggestion
  [sidebar: TinaCMSConfig['sidebar']](https://tinacms.org/docs/ui#enabling-the-user-interface)
```

Comment (kendallstrautman) [reply packages/gatsby-plugin-tinacms/README.md:43]: seeing if this is weird putting a link in the interface

File: packages/gatsby-plugin-tinacms/README.md [hunk 3]
```diff
@@ -36,20 +22,28 @@ module.exports = {
...
 }
 ```
 
-## Next Steps
+## Options
+
+```ts
+export interface GatsbyPluginTinacmsOptions {
+  enabled?: boolean
+  [sidebar: TinaCMSConfig['sidebar']](https://tinacms.org/docs/ui#enabling-the-user-interface)
```
Comment (kendallstrautman) [inline packages/gatsby-plugin-tinacms/README.md:43]:
```suggestion
  sidebar: TinaCMSConfig['sidebar']

Comment (kendallstrautman) [reply packages/gatsby-plugin-tinacms/README.md:43]: yep, its weird lol

**Pull request merged**